server/spec/test_integration_data_provider_esb.rb

```diff
@@ -49,9 +54,10 @@ def test_esb_with_good_uniqname
       @@yml = nil
     end.new
 
-    refute_nil(m,"create provider object")
-    classes = m.dataProviderESBCourse(@known_uniqname, @default_term, @security_file,@esb_application,@default_term)
-    assert_equal(200,classes.meta_status,'find classes for good uniqname')
+    @default_term = 2060
```

Comment:
This is not ideal, but since this is for testing I think this is OK until testing ESB APIs gets easier.